sqlite/src/comdb2build.c

```diff
@@ -404,15 +404,13 @@ int comdb2PrepareSC(Vdbe *v, Parse *pParse, int int_arg,
 int (*externalComdb2AuthenticateUserDDL)(void*, const char *tablename) = NULL;
 int (*externalComdb2CheckOpAccess)(void *) = 0;
 void * (*externalMakeAuthData) (const char *, int, int, int, void *);
-void * (*externGetAuthData) (void*) = 0;
 
 static int comdb2AuthenticateUserDDL(const char *tablename)
 {
      struct sqlclntstate *clnt = get_sql_clnt();
 
      if (gbl_uses_externalauth && externalComdb2AuthenticateUserDDL) {
-         if (!clnt->authdata && externGetAuthData)
-             clnt->authdata = externGetAuthData(clnt);
+         clnt->authdata = clnt->plugin.get_authdata(clnt);
```

Comment:
Can we introduce a wrapper for this along the lines of `int param_count(struct sqlclntstate *)` etc (in `sql.h`)

Reply:
done